src/Jackett.Common/Indexers/TorrentSyndikat.cs

```diff
@@ -94,160 +90,184 @@ public TorrentSyndikat(IIndexerConfigurationService configService, WebClient w,
-    }
+        public sealed override TorznabCapabilities TorznabCaps
+        {
+            get => base.TorznabCaps;
+            protected set => base.TorznabCaps = value;
+        }
 
         public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
         {
-            LoadValuesFromJson(configJson);
-            CookieHeader = "";
-
-            var result1 = await RequestStringWithCookies(CaptchaUrl);
-            var json1 = JObject.Parse(result1.Content);
-            var captchaSelection = json1["images"][0]["hash"];
+            ConfigData.LoadValuesFromJson(configJson);
+            var releases = await PerformQuery(new TorznabQuery());
 
-            var pairs = new Dictionary<string, string> {
-                { "username", configData.Username.Value },
-                { "password", configData.Password.Value },
-                { "captchaSelection", (string)captchaSelection },
-                { "submitme", "X" }
-            };
+            await ConfigureIfOK(
+                string.Empty, 
+                releases.Any(), 
+                () => throw new Exception("Could not find any releases"));
 
-            var result2 = await RequestLoginAndFollowRedirect(LoginUrl, pairs, result1.Cookies, true, null, null, true);
-
-            await ConfigureIfOK(result2.Cookies, result2.Content.Contains("/logout.php"), () =>
-            {
-                var errorMessage = result2.Content;
-                throw new ExceptionWithConfigData(errorMessage, configData);
-            });
-            return IndexerConfigurationStatus.RequiresTesting;
+            return IndexerConfigurationStatus.Completed;
         }
 
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
-            List<ReleaseInfo> releases = new List<ReleaseInfo>();
+            var releases = new List<ReleaseInfo>();
 
-            var searchString = query.GetQueryString();
-            var searchUrl = SearchUrl;
-            var queryCollection = new NameValueCollection();
+            var searchstring = query.GetQueryString();
+            var queryCollection = new NameValueCollection {{"apikey", ConfigData.Key.Value}};
 
-            queryCollection.Add("incldead", "1");
-            queryCollection.Add("rel_type", "0"); // Alle
+            queryCollection.Add("limit", "50"); // Default 30
+            //queryCollection.Add("ponly", true); // Torrents with products only
 
             if(query.ImdbID != null)
             {
-                queryCollection.Add("searchin", "imdb");
-                queryCollection.Add("search", query.ImdbID);
+                queryCollection.Add("imdbId", query.ImdbIDShort);
             } else {
-                queryCollection.Add("searchin", "title");
-                
-                if (!string.IsNullOrWhiteSpace(searchString))
+                if (!string.IsNullOrWhiteSpace(searchstring))
```

Comment:
`Else { Nothing }`. If no search string is provided we don't need to provide the `searchstring` query parameter, in that case it will just return the latest torrents. Purely optional.

Reply:
I explainded bad myself:
```
            if (query.ImdbIDShort != null)
            {
                queryCollection.Add("imdbId", query.ImdbIDShort);
            } else if (!string.IsNullOrWhiteSpace(searchstring))

            }
```

Reply:
Ah, thanks. Missed that one.